lib/db_constants.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
similar to the other PR, can we just rename this file `constants.js`, in the likely event we will end up with non-db related constants. Trying to contain file bloat here.

Reply:
Do you think that we shouldindividually bring the constants on to the `Db` constructor instead of using `Object.assign`? If we add new constants, it will be easy to accidentally expose them.